packages/components/src/segmented-control/segmented-control.tsx

```diff
@@ -0,0 +1,261 @@
+import {
+  Children,
+  cloneElement,
+  forwardRef,
+  isValidElement,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+
+import { cva } from 'cva'
+
+import type { ReactElement, ReactNode } from 'react'
+
+const tabContainerStyles = cva({
+  base: 'relative flex flex-1 items-center justify-center gap-0.5 rounded-10 p-0.5',
+  variants: {
+    type: {
+      grey: 'bg-neutral-10 blur:bg-neutral-80/5 blur:backdrop-blur-[20px] dark:bg-neutral-80 blur:dark:bg-white-5',
+      'dark-grey': 'bg-neutral-20 dark:bg-neutral-90',
+    },
+    size: {
+      '24': 'h-6',
+      '32': 'h-8',
+    },
+  },
+})
+
+const tabStyles = cva({
```

Comment:
If I get this correctly, this should be active tab styling? I'd name it like that.